src/sync/installation.ts

```diff
@@ -406,22 +417,36 @@ export const processInstallation =
 				}
 
 				// Continue sync when a 404/NOT_FOUND is returned
-				if (handleNotFoundErrors(err, job, nextTask)) {
+				if (isNotFoundError(err, job, nextTask)) {
 					const edgesLeft = []; // No edges left to process since the repository doesn't exist
 					await updateJobStatus(queues, job, edgesLeft, task, repositoryId);
 					return;
 				}
 
-				await subscription.update({ syncStatus: "FAILED" });
+				if (await booleanFlag(BooleanFlags.CONTINUE_SYNC_ON_ERROR, false, jiraHost)) {
+
+					logger.warn({ job, task: nextTask, err }, "Task failed, continuing with next task");
 
-				const host = subscription.jiraHost || "none";
-				logger.warn({ job, task: nextTask, err, jiraHost: host }, "Sync failed");
+					// marking the current task as failed
+					await subscription.updateRepoSyncStateItem(nextTask.repositoryId, getStatusKey(nextTask.task as TaskType), "failed");
 
-				job.sentry.setExtra("Installation FAILED", JSON.stringify(err, null, 2));
-				job.sentry.captureException(err);
+					statsd.increment(metricTaskStatus.failed, [`type: ${nextTask.task}`]);
```

Comment:
So we count here failed tasks, correct? 

Should we also count successful tasks?

If one task not equal to one repo, should we add a separate metric to count successfully sync & failed repos?